docs/changelog.rst

```diff
@@ -30,10 +30,11 @@ Bug fixes:
   such as `bandcamp_album_id` will be available on albums in addition to tracks.
   For albums already in your library, a re-import is required for the fields to be added.
   Such a re-import can be done with, in this case, `beet import -L data_source:=MusicBrainz`.
-* :doc:`plugins/autobpm`: Fix the ``TypeError`` where tempo was being returned
-  as a numpy array. Update ``librosa`` dependency constraint to prevent similar
+* :doc:`plugins/autobpm`: Fixed the ``TypeError`` where tempo was being returned
```

Comment:
I'd suggest to keep as it was since the rest of the changelog prefers 'Fix' over 'Fixed':

```fish
$ grep -i -e fix -e fixed docs/changelog.rst -o |
  tr '[:upper:]' '[:lower:]' |
  sort |
  uniq -c |
  sort -n

     62 fixed
    370 fix

```